README.md

```diff
@@ -83,7 +83,8 @@ Connaisseur can be integrated into a multitude of different systems, with a few
 1. **Requirements**: A few tools are needed in advance. You'll need `git` for getting the source code and `make` for a convenient setup. Since Connaisseur will be deployed as a container to a Kubernetes cluster, [docker](https://docs.docker.com/v17.09/engine/installation/linux/docker-ce/ubuntu/), [helm](https://helm.sh/docs/intro/install/) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) are required for building the image, generating a Deployment file and deploying it to the cluster. During the deployment, some certificates need to be created, which requires `openssl`. To install everything except docker, helm and kubectl, use
 
 ```bash
-sudo apt install git make openssl yq
+sudo apt install git make openssl
```

Comment:
Do we want to put -y options now that this is multiline?

Reply:
Hm, that doesn't help because you need to enter your passphrase, does it? Are you able to install `yq`via apt? It would be better to have it in one line, but apt wasn't able to locate `yq`...

Reply:
Would probably still put the `-y` so it works if run from elevated context